dali/base/dafdesc.cpp

```diff
@@ -3652,8 +3659,12 @@ static void doInitializeStorageGroups(bool createPlanesFromGroups)
         storage.set(globalConfig->addPropTree("storage"));
 
 #ifndef _CONTAINERIZED
-    if (createPlanesFromGroups && !storage->hasProp("planes"))
+    if (createPlanesFromGroups)
     {
+        // Remove old planes created from groups
+        while (storage->removeProp("planes[@fromGroup='1']"));
```

Comment:
minor style: For loops with empty bodies it is clearer to code as the following:

while (storage->removeProp("planes[@fromGroup='1']"))
{
}

because it makes it clear that the body is intentionally blank.  No need to change for this PR since it matches other examples.

Although having said that there should be a function that avoids the need for the loop.  @jakesmith Added HPCC-33048.